web-console/src/views/sql-view.scss

```diff
@@ -38,3 +38,26 @@
     flex: 1;
   }
 }
+
+.sql-view-explain-dialog {
```

Comment:
when you are starting a new top level css block, that is when you know it should really be its own file